nasa-gibs/worldview
# wv-2798: Fix references

## Description
Point all URLs to production. Information about the existence of non-production instances should not be available publicly. 

Fixes #wv-2798 .

## How To Test
- git checkout wv-2798
- npm ci
- npm run build
- npm run watch
- Confirm that OrbitTracks Aqua Ascending works properly
- Confirm that Geographic NRT layers work properly

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
